CMakeLists.txt

```diff
@@ -579,8 +596,11 @@ configure_file(
 
 add_subdirectory(tools/rc)
 
-export(TARGETS ${US_RCC_EXECUTABLE_TARGET}
-       FILE ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Targets.cmake)
+# Only export the target if we are not cross-compiling, because the target is only built during the host build
+if(NOT CMAKE_CROSSCOMPILING)
```

Comment:
I believe the intent was to cross-compile bundles only, which if fine. However, we might want to cross-compile *and* install the complete project in the future, such that an install tree can be packaged and used on the target platform to  develop native bundles (I am thinking of e.g. creating .deb or .rpm packages in the future). This means we would have to buid and install the cross-compiled version of usResouceCompiler as well.

Reply:
good point. I agree.